src/components/product/ProductList.component.tsx

```diff
@@ -8,11 +9,14 @@ import ProductCard from './ProductCard.component';
 import styles from './Product.module.css';
 
 const ProductList = () => (
-    <ul className={styles.product__list}>
-        {data.map((product: Product) => (
-            <ProductCard key={product.id} product={product} />
-        ))}
-    </ul>
+    <section className={styles.product__section}>
```

Comment:
Please correct border and searc text color 
<img width="971" alt="image" src="https://github.com/demasha0908/fe-react-2024/assets/98031768/9da731fd-9748-495c-a008-5abd7925ad2e">